framework_tvb/tvb/interfaces/web/controllers/flow_controller.py

```diff
@@ -662,7 +661,7 @@ def store_exploration_section(self, val_range, step, dt_group_guid):
         datatype_group_ob = ProjectService().get_datatypegroup_by_gid(dt_group_guid)
         operation_grp = datatype_group_ob.parent_operation_group
         operation_obj = OperationService.load_operation(datatype_group_ob.fk_from_operation)
-        parameters = json.loads(operation_obj.parameters)
+        parameters = {}
```

Comment:
Why keep this empty dict ?

Reply:
it is populated below and uses range1 and range2